modules/distributor/config.go

```diff
@@ -43,6 +45,12 @@ type Config struct {
 	factory func(addr string) (ring_client.PoolClient, error) `yaml:"-"`
 }
 
+type LogReceivedSpansConfig struct {
+	Enabled             bool `yaml:"enabled"`
+	IncludeAttributes   bool `yaml:"include_attributes"`
```

Comment:
This can become incredibly verbose, as cardinality in traces tends to be high. What do you think of changing it to a `[]string` so specific attributes can be selected instead? It can have a `all` option to just print everything.

Reply:
If we make it a boolean now and later want to change into `[]string` that would be a breaking change (or some nasty yaml handling). So perhaps it's better to add it now.
On the other hand, I'm in favour of keeping it simple and reducing the amount of configuration options. Cardinality isn't a concern for log fields but the volume of logs will be.

Reply:
@mapno should I modify it like this?

```yaml
log_received_spans:
    [enabled: <boolean> | default = false]
    [include_attributes: <list of string> | default = ] // To include all attributes use `all`]
    [filter_by_status_error: <boolean> | default = false]
```

I can also rename `include_attributes` to `include_all_attributes` so if in the future is decided to have the option to select just a few attributes we could use `include_attributes` as the list of attributes.

@kvrhdn indeed, if this feature is used in production without the error filter it can overwhelm the log engine.

Reply:
I like the proposal of using `include_all_attributes`. It's explicit and keeps the door open for extension in the future.

Reply:
done :)